src/schema.ts

```diff
@@ -273,44 +277,17 @@ export interface FlywheelConnectedEvent {
 }
 
 /**
- * Represents every channel that can be played on the vibraphone.
- *
- * We decided to start at one since the [[BassString]] type also
- * starts at one. We also decided to only represent playable
- * channels and not actual channels (two per note) because that is
- * an implementation detail of the physical MMX.
+ * All possible events relating to the machine itself.
  */
-export type VibraphoneChannel = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
+export type MachineEvent = MachineMuteEvent | MachineTempoEvent | FlywheelConnectedEvent;
 
-/**
- * Represents the state pertaining to the vibraphone.
- */
-export interface VibraphoneState {
-	/** Whether or not the vibrato is engaged */
-	vibratoEnabled: boolean;
```

Comment:
Isn't having 0 for the speed the same as not engaged?

Reply:
Physically the motor can be running but disconnected, this more accurately represents the machine.